mosquitto/socket/readme.md

```diff
@@ -0,0 +1,7 @@
+A local mosquitto server using `mosquitto/config/mosquitto.conf` will create a `mosquitto.sock` socket in this directory.
```

Comment:
It is common practice to add a `.gitkeep` file in a folder you want to keep around.